src/components/IdCard.js

```diff
@@ -0,0 +1,21 @@
+import './IdCard.css'
+
+
+function IdCard (props) {
+    return(
+      <section>
+      <div>
+      <img src = {props.picture} alt='id card'/>
+      </div>
+      <div>
+      <p>lastName={props.lastName}</p>
+      <p>firstName={props.firstName}</p>
+      <p>gender={props.gender}</p>
+      <p>height={props.height}</p>
+      <p>birth={props.birth}</p>
+      </div>
+      </section>
+    )     
+  }
```

Comment:
Si aprovechamos la desestructuración de objetos lo podemos hacer así:

```suggestion
function IdCard ({picture, lastName, firstName, gender, height, birth}) {
    return(
      <section>
      <div>
      <img src = {picture} alt='id card'/>
      </div>
      <div>
      <p>lastName={lastName}</p>
      <p>firstName={firstName}</p>
      <p>gender={gender}</p>
      <p>height={height}</p>
      <p>birth={birth}</p>
      </div>
      </section>
    )     
  }
```

Reply:
Super importante la IDENTACIÓN ! Hay que mantener el orden mientras se programa :wink: 

![image](https://user-images.githubusercontent.com/26923217/134134449-cc1a0025-8d9b-4c24-b31d-489afc0d4354.png)